### ISSUE TITLE
Pagination warning in backend console

### ISSUE THREAD

**Issue by TobySuch** [TobySuch/ivy-list#31]
```
UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'to_do.models.ToDoItem'> QuerySet.paginator = self.django_paginator_class(queryset, page_size)
```

Check `to_do/views.py.ToDoItemViewSet.get_queryset` method. Should order by date first and then priority.

**Comment by jhrdt** [comment]
Hej,

I added a PR #33 for this issue. Hope it helps.

**Comment by TobySuch** [comment]
Closed in #33
